modules/simple_forms_api/spec/requests/simple_forms_api/v1/simple_forms_spec.rb

```diff
@@ -42,7 +42,8 @@
     allow(SimpleFormsApiSubmission::MetadataValidator).to receive(:validate)
   end
 
-  describe '#submit' do
+  # TODO: Note to fix these tests for the team that manages them
+  xdescribe '#submit' do
```

Comment:
I had a task to skip this test. You got to it before me! Thanks! Try this:

```suggestion
  describe '#submit', skip: 'Flakey test' do
```